pkgs/development/python-modules/mutmut/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitHub
+, buildPythonApplication
+, click
+, glob2
+, parso
+, pony
+, junit-xml
+, pythonOlder
+, testVersion
+}:
+
+let self = buildPythonApplication rec {
+  pname = "mutmut";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "boxed";
+    rev = version;
+    hash = "sha256-G+OL/9km2iUeZ1QCpU73CIWVWMexcs3r9RdCnAsESnY=";
+  };
+
+  postPatch = ''
+    sed -i 's/junit-xml==1.8/junit-xml==1.9/' requirements.txt
```

Comment:
Is there any reason to prefer this instead of `substituteInPlace`?

Reply:
No, I initially wrote this as a personal thing before I decided to upstream it. I'll change it